### DISCUSSION TITLE
Finetune embedding in new domain with only 2 class

### DISCUSSION THREAD

**Discussion by dungnguyen98** [speechbrain/speechbrain#1880] [Q&A]
Thanks for great repository.
I am facing a problem. My data is about calls between an agent and a customer (every files have only 2 speakers).

I use segmentation model to get voice zones. However, when I put these voice segments through embedding to get embedding of each zone, then perform clustering (2 clusters), I encountered some confusion. 
How can I fine tune embedding for my problem ?

**Comment by anautsch** [comment]
Hi @dungnguyen98 fine-tuning is described in our tutorial:
https://colab.research.google.com/drive/1LN7R3U3xneDgDRK2gC5MzGkLysCWxuC3?usp=sharing

Other tutorials might help you also:
https://speechbrain.github.io/tutorial_advanced.html

Are you extracting embeddings from mono or stereo files?
Please explain more about your confusion.
